ios/Classes/Module/FacebookModule.m

```diff
@@ -965,7 +975,7 @@ + (FBSDKSharePhotoContent *_Nonnull)sharePhotoContentFromDictionary:(NSDictionar
   NSMutableArray<FBSDKSharePhoto *> *nativePhotos = [NSMutableArray arrayWithCapacity:photos.count];
 
   for (NSDictionary<NSString *, id> *photoDictionary in photos) {
-    id photo = photoDictionary[@"photo"];
+    id photo = photoDictionary[@"image"];
```

Comment:
This would be a breaking change in our implementation. You just need to adjust our [example](https://github.com/appcelerator-modules/ti.facebook/blob/85db73169b0211541096b2b58d8161e957113bcc/example/facebook_share.js#L64) to use  the`photo` property, as stated in the [docs](https://docs.appcelerator.com/platform/latest/#!/api/SharePhotoContentPhotoParams)

Reply:
Changes are done as per document.